web/src/compose_fade.ts

```diff
@@ -15,57 +16,78 @@ import * as util from "./util";
 
 // TODO/TypeScript: Move this to message_list_view.js when it's migrated to TypeScript.
 export type MessageContainer = {
-    background_color: string;
-    date_divider_html?: string;
-    edited_alongside_sender: boolean;
-    edited_in_left_col: boolean;
-    edited_status_msg: boolean;
+    background_color?: string;
+    date_divider_html: string | undefined;
+    edited_alongside_sender?: boolean;
+    edited_in_left_col?: boolean;
+    edited_status_msg?: boolean;
```

Comment:
I think the 3 `edited_*` booleans here are are dead code that should have been removed in ea2b659314459b571ac0fa627fd2617c2cce4f3c.

Reply:
Yup looks like it, adding a prep commit for that

Reply:
It seems just as fine to fix in a follow-up commit.